osuosl/climesync
# Added option to output TimeSync data to a CSV file

fixes issue #70 
## Changes in this PR.


- [X] After running a get_\* command, the user is asked if they want to save the result into a CSV file
## Testing this PR.


1. Run `nosetests`
2. Run the program and run the command `gt` to get times without entering any filter parameters.
3. When asked if you want to save the result to a CSV file, type `y` and enter a file name
4. Open that file in your favorite text editor and see that the values have been put into the file in CSV format

@osuosl/devs
